### PR TITLE
Fix blender crash redux

### PR DESCRIPTION
**Pull request by ZappoMan**
This fixes the original bugsplat 1899 that was crashing in hasBlendedMeshes(), but also doesn't cause problems with new crashes on multiple avatars or when changing your own avatar.

### CODE REVIEW

**Comment by ZappoMan** [comment]
Testing Notes:
1) Run this build, use an avatar with mouth blendshapes (Albert or Will)
2) make sure you have mirror or mini-mirror visible, talk, you should see blendshapes for the mouth
3) Change your avatar, verify it doesn't crash
4) get multiple people to join you in the domain, and make sure you can see their blendshapes properly when they talk.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 2389295217e345bf48fbb030bb1267d03fa1f691 available at: https://gist.github.com/a8743e97b29bda14ca70<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7411.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7411.dmg

**Comment by PhilipRosedale** [comment]
Tested and works as described with two machines in playa.  all blendshapes visible, avatars chnaged, no crash.

File: libraries/render-utils/src/Model.cpp [hunk 1]
```diff
@@ -1295,11 +1293,15 @@ void ModelBlender::setBlendedVertices(const QPointer<Model>& model, int blendNum
...
-            return;
+    while (!_modelsRequiringBlends.empty()) {
+        auto fistItem = _modelsRequiringBlends.begin();
+        if (fistItem != _modelsRequiringBlends.end()) {
+            _modelsRequiringBlends.erase(fistItem);
+            ModelPointer nextModel = fistItem->lock();
+            if (nextModel && nextModel->maybeStartBlender()) {
+                _pendingBlenders++;
+                return;
+            }
```
Comment (AndrewMeadows) [inline libraries/render-utils/src/Model.cpp]: Is this a typo?  `fistItem` should be `firstItem`?

**Comment by hifi-gustavo** [comment]
Build console output for commit: d28db7bec6b045da5efd975ef6db193971f388f6 available at: https://gist.github.com/a8743e97b29bda14ca70<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7411.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7411.dmg

**Pull request merged**